tests/TokenizerTest.php

```diff
@@ -7,9 +7,39 @@
 use Doctrine\SqlFormatter\Tokenizer;
 use PHPUnit\Framework\Attributes\DoesNotPerformAssertions;
 use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+
+use function array_filter;
+use function preg_match;
+use function strtoupper;
 
 final class TokenizerTest extends TestCase
 {
+    /**
+     * @param 'reserved'|'reservedToplevel'|'reservedNewline' $propertyName
+     *
+     * @return list<string>
+     */
+    private function getTokenizerList(string $propertyName): array
+    {
+        $tokenizerReflClass = new ReflectionClass(Tokenizer::class);
+        /** @var list<string> $res */
+        $res = $tokenizerReflClass->getProperty($propertyName)->getDefaultValue();
+
+        return $res;
+    }
+
+    public function testKeywordsReservedAreUpperCasedAndWithoutWhitespace(): void
+    {
+        $tokenizerReserved = $this->getTokenizerList('reserved');
+
+        $kwsDiff = array_filter($tokenizerReserved, static function ($v) {
+            return $v !== strtoupper($v) || preg_match('~\s~', $v) !== 0;
+        });
+
+        self::assertSame([], $kwsDiff);
+    }
```

Comment:
What is this method supposed to test? This is checking the format of the reserved keywords.

Reply:
All reserved keywords must be a single word in upper case (so we can rely on it when comparing againts other UC string).